tests/test_parse.py

```diff
@@ -0,0 +1,17 @@
+from morphocut import Pipeline
+from morphocut.str import Parse
+
+
+#TODO: Not working. Still under development
+def test_Parse():
+    pattern = "This is a {}"
```

Comment:
Parse's output is a dict of name: matched value. You used an anonymous format expression.

After rebasing onto upstream/master, you should get a ParseWarning about that. Let me know if not.

You should also assert (in a separate step) that this warning in fact is issued. See https://docs.pytest.org/en/latest/warnings.html

Reply:
https://trello.com/c/0Ez00NkP/11-rebase-feature-branch-onto-upstream-master